src/schoology-extractor/data_extractor/lib/export_data.py

```diff
@@ -0,0 +1,9 @@
+import pandas as pd
+
+
+def tocsv(data, output_path):
```

Comment:
Please add guard assertions to make sure that neither argument is `None`.

Reply:
And think about where appropriate in other functions below. I'm not going to comment on every one of them.